pkgs/development/python-modules/cmsdials/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pydantic_1,
+  pythonRelaxDepsHook,
+  requests,
+  typing-extensions,
+  pandas,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "cmsdials";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cms-DQM";
+    repo = "dials-py";
+    rev = "v${version}";
+    hash = "sha256-664FvPdZFU5eTcMadnHfs0ZeiKqTH+Gy2bnhIFQxU2o=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pythonRelaxDepsHook
+  ];
```

```suggestion
  build-system = [ poetry-core ];
```
We no longer need to add `pythonRelaxDepsHook`.
https://github.com/NixOS/nixpkgs/pull/319573